packages/bezier-react/src/components/Modals/ConfirmModal/ConfirmModal.test.tsx

```diff
@@ -0,0 +1,241 @@
+/* External dependencies */
+import React from 'react'
+import { isInaccessible } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+/* Internal dependencies */
+import { render } from 'Utils/testUtils'
+import { ConfirmModal } from './ConfirmModal'
+import { ConfirmModalContent } from './ConfirmModalContent'
+import { ConfirmModalHeader } from './ConfirmModalHeader'
+import { ConfirmModalFooter } from './ConfirmModalFooter'
+import { ConfirmModalTrigger, ConfirmModalClose } from './ConfirmModalHelpers'
+import { ConfirmModalProps, ConfirmModalContentProps } from './ConfirmModal.types'
+
+const TRIGGER_TEXT = 'Open'
+const CANCEL_TEXT = 'Cancel'
+const ACTION_TEXT = 'Action'
+const TITLE_TEXT = 'Title'
+const DESCRIPTION_TEXT = 'Description'
+
+describe('ConfirmModal', () => {
+  const renderModal = ({
+    modalProps,
+    modalContentProps,
+  }: {
+    modalProps?: ConfirmModalProps
+    modalContentProps?: ConfirmModalContentProps
+  } = {}) => render(
+    <ConfirmModal {...modalProps}>
+      <ConfirmModalTrigger>
+        <button type="button">{ TRIGGER_TEXT }</button>
+      </ConfirmModalTrigger>
+      <ConfirmModalContent {...modalContentProps}>
+        <ConfirmModalHeader
+          title={TITLE_TEXT}
+          description={DESCRIPTION_TEXT}
+        />
+        <ConfirmModalFooter
+          rightContent={(
+            <>
+              <ConfirmModalClose>
+                <button type="button">{ CANCEL_TEXT }</button>
+              </ConfirmModalClose>
+              <ConfirmModalClose>
+                <button type="button">{ ACTION_TEXT }</button>
+              </ConfirmModalClose>
+            </>
+          )}
+        />
+      </ConfirmModalContent>
+    </ConfirmModal>,
+  )
+
+  let user: ReturnType<typeof userEvent.setup>
+  let renderOpenedModal: typeof renderModal
+
+  beforeEach(() => {
+    user = userEvent.setup({
+      /**
+       * To prevent `pointer-events: none` error.
+       * @see https://testing-library.com/docs/user-event/options#pointereventscheck
+       * PointerEventsCheckLevel.Never
+       */
+      pointerEventsCheck: 0,
+    })
```

Comment:
```suggestion
    import userEvent, { PointerEventsCheckLevel } from '@testing-library/user-event'

    user = userEvent.setup({
      /**
       * To prevent `pointer-events: none` error.
       * @see https://testing-library.com/docs/user-event/options#pointereventscheck
       * PointerEventsCheckLevel.Never
       */
      pointerEventsCheck: PointerEventsCheckLevel.Never,
    })
```

Reply:
오 이걸 export 해주고 있었군요 👍